lisp/ess-custom.el

```diff
@@ -551,19 +551,25 @@ buffer or end chunks etc.")
 
 (defcustom ess-S-assign " <- "
   "String used for left assignment in all S dialects.
-Used by \\[ess-smart-S-assign]."
+Used by \\[ess-insert-S-assign]."
   :group 'ess-S
   :type 'string)
 
-(defcustom ess-smart-S-assign-key "_"
-  "Key used by `ess-smart-S-assign'.
-Should be nil or a \"simple\" key, in other words no key
-modifiers.
+(defcustom ess-assign-list '(" <- " " <<- " " = " " -> " " ->> ")
+  "List of assignment operators.
+`ess-cycle-assignment' uses this list.  These strings must
```

Comment:
Must they be surrounded by spaces? Can it be `'(" <- " "_")`?

Reply:
Kinda. If you have `<-` then `<<-`, the regex messes up. I figured that we should surround them with spaces anyway since `ess-S-assign` is also surrounded by spaces.

Reply:
Is regexp needed? Can we match strings literally instead?

Reply:
Matching literally has the same problem, right? In that `<-` will match  `<<-`?